nextcord/ext/commands/bot.py

```diff
@@ -694,7 +694,13 @@ def _load_from_module_spec(
 
         extras = extras or {}
         try:
-            setup(self, **extras)
+            if asyncio.iscoroutinefunction(setup):
+                try:
+                    asyncio.create_task(setup(self, **extras))
+                except RuntimeError:
+                    raise RuntimeError(f"Seems like this extension is having an async setup function. Please read our FAQ here: https://nextcord.readthedocs.io/faq.html#how-do-i-make-my-setup-function-a-coroutine-and-load-it")
```

Comment:
``Seems like this extension has an async setup function. Please read our FAQ here: https://nextcord.readthedocs.io/faq.html#how-do-i-make-my-setup-function-a-coroutine-and-load-it``

Alternately:
``Looks like your attempting to load an asynchronous setup function incorrectly. Please read our FAQ here: https://nextcord.readthedocs.io/faq.html#how-do-i-make-my-setup-function-a-coroutine-and-load-it``


Further, please wrap this to be multiple lines, not just one big line

Reply:
done.